lib/committee/test/methods.rb

```diff
@@ -17,14 +17,21 @@ def assert_request_schema_confirm
         schema_validator.request_validate(request_object)
       end
 
-      def assert_response_schema_confirm
+      def assert_response_schema_confirm(expected_status = nil)
```

Comment:
I thought it would be better to set a default value, but since it is a feature that does not exist originally, there is no point in maintaining compatibility, and this method of setting by the user is good.